src/HashBus.WebApi/MostHashtaggedModule.cs

```diff
@@ -35,8 +35,9 @@ public MostHashtaggedModule(IRepository<string, IEnumerable<Hashtag>> hashtags)
                 {
                     Entries = entries,
                     Count = trackHashtags.Count,
-                    Since = trackHashtags.Min(hashtag => hashtag.HashtaggedAt),
-                    LastActivityDateTime = trackHashtags.Max(hashtag => hashtag.HashtaggedAt),
+
+                    Since = trackHashtags.Any() ? trackHashtags.Min(hashtag => hashtag.HashtaggedAt) : (DateTime?)null,
+                    LastActivityDateTime = trackHashtags.Any() ? trackHashtags.Max(hashtag => hashtag.HashtaggedAt) : (DateTime?)null

```

Comment:
Also another nitpick, but this line has the trailing comma removed. It's a style thing, but personally I prefer to have every line in an initialiser to have a trailing comma so they are consistent, can be moved around, don't have to be altered when a new line is added to the end of the list, etc.